### ISSUE TITLE
Account name change reflects on other accounts

### ISSUE THREAD

**Issue by housemobile** [ethereum/mist#1722]
I have multiple accounts.

I named the accounts as follows:

house
mobile
test
example

On the left side menu they show up properly. However, when clicking on each account the account name is prefixed by the latest changed account i made. eg.

examplehouse
examplemobile
exampletest
exampleexample

#### System information
``````
Version: `0.0.0`
OS & Version: windows/linux/osx
Node type: `eth/geth(default)`
``````

#### Please check the **already existing** issues to keep duplicates at a minimum.
Furthermore several **work-arounds** have been collected in the [Mist-Troubleshooting-Guide](https://github.com/ethereum/wiki/wiki/Mist-Troubleshooting-Guide).

If possible add the following to your report:

- Screenshots

- Check the console, of Mist (`CTRL/CMD + ALT + i`) and take a screenshot

- Log files 
  - go to `menu -> accounts -> backup -> application data`
  - zip and upload `node.log` and all other `node.log.X` files

**Comment by luclu** [comment]
@housemobile I tried to reproduce the issue, but failed. These are my parameters:
- Mist 0.8.9
- renamed three accounts during one session
  - the names reflect on the sidebar
  - after restarting they are still correct

Could you please provide the following:
- version of Mist
- screenshots from the console
- log files

**Comment by housemobile** [comment]
Running 0.8.9 on win64 on Windows 7 

I just did it again. Renamed as follows:

Example 1
Second
Third

After renaming the last one as 'Third' I clicked on the other addresses from the sidebar and they show as follows:

http://i.imgur.com/jUr69Gu.png

SecondThird (no screen for that one)

http://i.imgur.com/TmosFW3.png

Renaming like this: http://i.imgur.com/YzxzJ3y.png which I assume is the only way to rename anyway. 


It only does it when clicking on the other addresses from the sidebar after renaming. Once you click on Wallets, it fixes everything to how it should be.
